icons/testrail/testrail-plain.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
+  <path fill="#50d18d" d="M 37.706 127.428 L 22.31 112.649 C 21.531 111.919 21.531 110.713 22.31 109.983 L 37.718 95.213 C 38.479 94.466 39.735 94.466 40.498 95.213 L 55.906 109.993 C 56.684 110.723 56.684 111.929 55.906 112.659 L 40.498 127.428 C 39.735 128.176 38.479 128.176 37.718 127.428 L 37.706 127.428 Z M 62.608 103.542 L 47.2 88.762 C 46.428 88.03 46.428 86.836 47.2 86.106 L 62.608 71.327 C 63.371 70.58 64.627 70.58 65.388 71.327 L 80.786 86.106 C 81.564 86.836 81.564 88.042 80.786 88.772 L 65.388 103.542 C 64.627 104.282 63.376 104.282 62.615 103.542 L 62.608 103.542 Z M 87.512 79.655 L 72.114 64.874 C 71.341 64.144 71.341 62.95 72.114 62.219 L 87.517 47.44 C 88.28 46.693 89.536 46.693 90.297 47.44 L 105.695 62.219 C 106.467 62.95 106.467 64.144 105.695 64.874 L 90.285 79.655 C 89.524 80.4 88.268 80.4 87.507 79.655 L 87.517 79.655 L 87.512 79.655 Z M 38.174 80.102 L 22.765 65.333 C 21.993 64.602 21.993 63.403 22.765 62.673 L 38.174 47.887 C 38.934 47.14 40.191 47.14 40.953 47.887 L 56.361 62.668 C 57.134 63.398 57.134 64.592 56.361 65.323 L 40.953 80.114 C 40.191 80.859 38.934 80.859 38.174 80.114 L 38.174 80.102 Z M 63.076 56.215 L 47.667 41.446 C 46.89 40.716 46.89 39.51 47.667 38.779 L 63.076 24 C 63.838 23.253 65.094 23.253 65.855 24 L 81.253 38.779 C 82.026 39.51 82.026 40.705 81.253 41.436 L 65.855 56.215 C 65.094 56.957 63.843 56.957 63.082 56.215 L 63.082 56.205 L 63.082 56.21 L 63.076 56.215 Z M 38.651 32.809 L 23.232 17.986 C 22.453 17.256 22.453 16.05 23.232 15.319 L 38.651 0.572 C 39.413 -0.175 40.67 -0.175 41.43 0.572 L 56.839 15.351 C 57.618 16.082 57.618 17.288 56.839 18.018 L 41.42 32.765 C 40.658 33.512 39.401 33.512 38.641 32.765 L 38.651 32.797 L 38.651 32.809 Z" style=""/>
+</svg>
```

Comment:
- File should be optimized e.g. using [SVGOMG](https://jakearchibald.github.io/svgomg/). This will remove unneeded svg element, and reduce the file size.

- The final SVG file, should be on a single line, with a empty line after

- The name for this file should be `testrail-original`, you can then use the alias for `testrail-plain` like this:

```json
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```